source/codemirror-tscriptmode.js

```diff
@@ -19,6 +19,7 @@ CodeMirror.defineMode("text/tscript", function(config, parserConfig)
 			"then": true,
 			"else": true,
 			"for": true,
+			"in": true,
```

Comment:
I see why you want to do that, but `in` is not a keyword (I changed that at some point). For example, a variable can be called in:

    var in = 7;
    print(in);

That's why I think that this highlighting would be confusing. Alternatively, we could discuss whether `in` should become a keyword or not.

Reply:
I think that `in` might be useful to be considered a keyword, because the `for`-syntax uses it as a syntactic element. I added `in` to the highlighting previously, because `in` might be highlighted like a keyword in a `for`-loop. `in` might also be considered as the operator to check if an element is in a container, similar to python. But this is not really needed, because `Array` and `Dictionary` have the function `has`, that already is very concise.